README.md

```diff
@@ -16,19 +16,20 @@ Tasks sent from a synchronous context to an asynchronous context in Swift Concur
 @MainActor
 func testMainActorTaskOrdering() async {
     actor Counter {
-        func increment() -> Int {
+        func incrementAndAssertCountEquals(_ expectedCount: Int) {
             count += 1
-            return count
+            let incrementedCount = count
+            XCTAssertEqual(incrementedCount, expectedCount) // often fails
```

Comment:
> Curious why we need/want to change to this version of the code that no longer returns the new count.

In this example it doesn't matter much, but in later examples where we have code like the following it becomes necessary and I didn't want to change examples too much:
```
    for iteration in 1...100 {
        counter.incrementAndAssertCountEquals(iteration)
    }
```

If we used an `increment` that would return, we'd end up with:
```
    for iteration in 1...100 {
        let count = await counter.increment()
        XCTAssertEqual(count, iteration) // this test will succeed since we `await` the return value of `increment` each time
    }
```